controllers/apply/get-advice/form.js

```diff
@@ -412,6 +412,137 @@ module.exports = function({ locale = 'en', data = {} } = {}) {
         });
     }
 
+    function fieldOrganisationType() {
+        return new RadioField({
+            name: 'organisationType',
+            label: localise({
+                en: 'What type of organisation are you?',
+                cy: 'Pa fath o sefydliad ydych chi?'
+            }),
+            options: [
+                {
+                    value: 'unregistered-vco',
```

Comment:
I'm deliberately not using any constants here or sharing the code from awards for all a) we don't need to depend on organisation type for sub-questions here and b) the actual list may diverge from the simple product (e.g. different options based on country, and some different valid types) so doesn't make sense to share anything.